packages/wallet-backend-update/src/rapyd/service.ts

```diff
@@ -0,0 +1,196 @@
+import { NotFound } from '@/errors'
+import { User } from '@/user/model'
+import crypto from 'crypto'
+import { Logger } from 'winston'
+import { RapydClient } from './rapyd-client'
+
+interface RapydServiceDependencies {
+  rapyd: RapydClient
+  logger: Logger
+}
+
+type VerifyIdentityParams = {
+  userId: string
+  documentType: string
+  frontSideImage: string
+  frontSideImageType: string
+  faceImage: string
+  faceImageType: string
+  backSideImage?: string
+  backSideImageType?: string
+}
+
+type CreateWalletParams = {
+  firstName: string
+  lastName: string
+  address: string
+  city: string
+  country: string
+  zip: string
+  email: string
+  id: string
+  phone?: string
+}
+
+export class RapydService {
```

Comment:
We should create a Rapyd interface.

```suggestion
export class RapydService implements IRapydService {
```